### ISSUE TITLE
Model Training & Tuning

### ISSUE THREAD

**Issue by alexander-harmaty** [alexander-harmaty/Breast-Cancer-Prognosis-Prediction#10]
• Train the joint model using the fused features.
• Conduct experiments to tune hyperparameters (learning rate, batch size, etc.).
